app/assets/javascripts/application.js

```diff
@@ -12,4 +12,9 @@
 //
 //= require rails-ujs
 //= require turbolinks
+//= require bootstrap
+//= require jquery
+//= require jquery_ujs
+//= require underscore
```

Comment:
For what underscore?

Reply:
In gmaps4rails gem instruction need underscore